notebooks/community/vision_ai/video_warehouse_sdk.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<p>For subfolder naming, change vision_ai to vision</p><p> </p><p>Add workbench link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
changed folder and add a link. I'll go ahead test with workbench as well. mark this resolved.